docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -77,7 +78,7 @@ import { createClient } from "@liveblocks/client";
 import { decode } from "base-64";
 
 const client = createClient({
-  /* ... other options ... */
+  // Other options
```

Comment:
I do feel the `/* ... */` style block works better here, because it's a placeholder for where code needs to be written. The `// Other options` line comment is something I would read as describing the lines below it, which is not the case here.